src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ChannelMemberRepository.cs

```diff
@@ -70,6 +70,24 @@ public async Task<ChannelMember> GetChannelMemberAsync(Guid memberId, Guid chann
             }
         }
 
+        public async Task<ChannelMember> GetChannelMemberWithMemberDetailsAsync(Guid memberId, Guid channelId)
+        {
+            using (var connection = _sqlConnectionFactory.CreateConnection())
+            {
+                var sqlQuery = $@"SELECT *
+                                 FROM ChannelMembers cm
+                                 INNER JOIN Members m ON m.{nameof(Member.Id)} = cm.{nameof(ChannelMember.MemberId)}
+                                 WHERE {nameof(ChannelMember.ChannelId)} = @{nameof(channelId)} AND {nameof(ChannelMember.MemberId)} = @{nameof(memberId)}";
```

Comment:
I think that we need to use here the following string:
WHERE cm.{nameof(ChannelMember.ChannelId)} = @{nameof(channelId)} AND cm.{nameof(ChannelMember.MemberId)} = @{nameof(memberId)}";

Reply:
done